robot/rover/comms/src/Serial.cpp

```diff
@@ -0,0 +1,37 @@
+#include "Serial.h"
+
+namespace internal_comms {
+
+    void startSerial(uint8_t rxPin, uint8_t txPin, long baudRate)
+    {
+        pinMode(rxPin, INPUT);
+        pinMode(txPin, OUTPUT); 
```

Comment:
We don't need this because `Serial` defines the pins already.
On teensies we have Serial which is usb and then Serial# where # is the uart port

Reply:
keep in mind that this needs to work with the PDS which doesn't use a teensy.